cms/core/permission_testers.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from wagtail.models import PagePermissionTester
+
+
+class BasePagePermissionTester(PagePermissionTester):
+    """Base class for page permission testers in the CMS."""
+
+    def can_add_subpage(self) -> bool:
```

Comment:
Have we tested if you can create Welsh first by copying a page?.

Reply:
Yes, unfortunately it will let you open the copy menu for Welsh pages, but when you try to actually click "Copy this page" it fails with message `You do not have permission to copy to page "<destination>" `, so the `can_add_subpage` test is being called.

Reply:
The drawback of this approach is the fast that they can't copy. We'll revisit if they need to copy Welsh pages, and it is a pressing issue.

I also suggest updating the RBAC matrix on confluence to suggest that they can't copy Welsh pages.